source/class/qx/ui/menu/Manager.js

```diff
@@ -597,6 +598,7 @@ qx.Class.define("qx.ui.menu.Manager",
             this._onKeyPressRight(menu);
             break;
 
+          case "Tab":
```

Comment:
Enhancement

Reply:
Kinda needed, but could be solved in another way. The menu did not disappear when the opener button blurred. This could have been handled by a blur listener, but this change had the side effect of the enhancement i added to the selectbox: Tab returns focus to the opener button